election/client/nodejs/Dockerfile

```diff
@@ -1,3 +1,5 @@
-FROM node:0.12
+FROM node:6.3-slim
```

Comment:
> Is this basically latest?

Yes, 6.3 is the latest stable version (check [this](https://nodejs.org/en/download/releases/) to see how old is 0.12 )